user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Creating_GQI_query.md

```diff
@@ -47,3 +47,5 @@ Please note the following regarding query data input:
 - From DataMiner 10.2.0/10.1.4 onwards, if a row is selected in a table component that uses a query feed, any view, service, or element linked to that row is also exposed as a feed. In practice, this means that a feed will be available in the *feeds* section of the data pane that will change based on the selection in the table.
 
 - From DataMiner 10.2.0/10.1.5 onwards, you can link GQI nodes that require a time range selection to a time range feed by selecting the *From feed* checkbox.
+
+- From DataMiner 10.3.0 [CU9]/10.3.12 onwards<!--RN 37505-->, the culture used in your GQI query matches the client culture of the web app. Prior to DataMiner 10.3.0 [CU9]/10.3.12 onwards, the invariant culture is used.
```

Comment:
@skyline-rge, thanks for the feedback! Is this better?

Reply:
I think this is good.